masonry/src/widget/image.rs

```diff
@@ -28,7 +28,7 @@ use crate::{
 /// than the image size).
 pub struct Image {
     image_data: ImageBuf,
-    fill: FillStrat,
+    fill: ObjectFit,
```

Comment:
Should we rename this field? I'd suggest either `fit` or `object_fit`?

Reply:
Yes we should rename it. 
`object_fit` is ok for the moment, anyway we can always rename it in the future